app/src/main/java/org/jellyfin/androidtv/util/RecommendedServerIssueExtensions.kt

```diff
@@ -0,0 +1,59 @@
+package org.jellyfin.androidtv.util
+
+import android.content.Context
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.auth.ServerRepository
+import org.jellyfin.sdk.api.client.exception.ApiClientException
+import org.jellyfin.sdk.discovery.RecommendedServerIssue
+import javax.net.ssl.SSLHandshakeException
+
+private fun ApiClientException.getFriendlyMessage(context: Context): String? = when (cause) {
+	is SSLHandshakeException -> context.getString(R.string.server_issue_ssl_handshake)
+	else -> null
```

Comment:
Could we print something like, "Unknown connection issue" here?

Reply:
No because we want to ignore other exception kinds here, I only used the when statement in case we want to match other exceptions in the future but that's unlikely.